examples/api-routes-apollo-server-and-client/lib/with-apollo-client.js

```diff
@@ -0,0 +1,64 @@
+import React from 'react'
+import initApollo from './init-apollo'
+import Head from 'next/head'
+import { getDataFromTree, ApolloProvider } from 'react-apollo'
+
+export default App => {
+  return class Apollo extends React.Component {
+    static displayName = `WithData(${App.displayName})`
+    static async getInitialProps (ctx) {
+      const { Component, router } = ctx
```

Comment:
You're not using `_app` but these are `_app` only props, because you'r'e using the page you should only need the component received in the exported function.